### PR TITLE
I2939 srop mentor display fixes

### LINKED ISSUE #2939: Fix SROP mentor display
# Problem
The images on this page don't look quite right:
Screen Shot 2021-01-26 at 12 57 07 PM

# Steps to reproduce
After the next build completes, this should be viewable at https://grad.prod.drupal.uiowa.edu/dei/srop/mentors

# Proposed solution
TBD

### PR DESCRIPTION
**Pull request by joewhitsitt**
Resolves #2939 

This fixes the reported issue but also addresses the issue comments.

Also moves the thesis defense stuff to grad_core where it is supposed to be. Oops.

**Before**:
Screen Shot 2021-01-27 at 12 14 27 PM

**After**:
Screen Shot 2021-01-27 at 12 14 51 PM

## Test

- View https://grad.local.drupal.uiowa.edu/dei/srop/mentors with stage file proxy on if you don't have files synced.
- Confirm that multiple positions are comma separated on the teaser and not on the full person node.
- Go to https://internationalighn.local.drupal.uiowa.edu and http://admissions.local.drupal.uiowa.edu/people/jessica-klein and confirm multi-value fields are comma-separated (twig debug should reference a uids_base location instead of the site's core module)

### CODE REVIEW

File: docroot/sites/grad.uiowa.edu/modules/grad_core/templates/node--mentor--teaser.html.twig [hunk 1]
```diff
@@ -71,85 +71,43 @@
...
+{{ attach_library('classy/node') }}
 
-<h3>{{ title }}</h3>
-
-{% set person_card_view = {
+{% set mentor_card = {
   'attributes': attributes.addClass(classes),
-  'headline_class': 'card__title headline',
+  'card_text': content|without('title', 'field_image', 'field_person_position', 'field_person_email', 'field_person_phone', 'links'),
+  'card_subtitle': content.field_person_position.0,
```
Comment (bspeare) [inline docroot/sites/grad.uiowa.edu/modules/grad_core/templates/node--mentor--teaser.html.twig]: @joewhitsitt Do we need to change this back to `content.field_person_position` to allow for multiple titles? I had forgotten about that part when I made my template comment in the issue.

Comment (joewhitsitt) [reply docroot/sites/grad.uiowa.edu/modules/grad_core/templates/node--mentor--teaser.html.twig]: changing back to `content.field_person_position` strings them together on the same line. do you know of an example where we pass a multiple value field to a card component?

Comment (pyrello) [reply docroot/sites/grad.uiowa.edu/modules/grad_core/templates/node--mentor--teaser.html.twig]: Should this be resolved in the field template for `field_person_position`?

Comment (bspeare) [reply docroot/sites/grad.uiowa.edu/modules/grad_core/templates/node--mentor--teaser.html.twig]: I think that's a good idea.

Comment (joewhitsitt) [reply docroot/sites/grad.uiowa.edu/modules/grad_core/templates/node--mentor--teaser.html.twig]: comma separated for the teaser

**Comment by bspeare** [comment]
@joewhitsitt is it worth adding https://github.com/uiowa/uiowa/blob/main/docroot/sites/admissions.uiowa.edu/modules/admissions_core/templates/field--comma-separated.html.twig to `uids_base` and then we can just `{% include '@uids_base/field/field--comma-separated.html.twig' %}` in field templates for future use?

**Comment by bspeare** [comment]
Pulling down to test.

**Comment by bspeare** [comment]
Looks like degree is displaying, did we want that?

Screen Shot 2021-01-27 at 3 20 57 PM

**Comment by joewhitsitt** [comment]
forgot to export the latest teaser config

**Comment by bspeare** [comment]
Do you mind if we add:

```
 [class*="card__"] & {
    font-size: inherit;
    line-height: inherit;
  }
``` 

To https://github.com/uiowa/uiowa/blob/main/docroot/themes/custom/uids_base/scss/fields/fields.scss#L40?  It would help these person cards using field templates retain their card styling.

**Comment by joewhitsitt** [comment]
>Do you mind

I do not. Do you have this branch local? I keep switching branches. I can get to it later if you can't.

**Review by bspeare — APPROVED** [review]
Tested on internationalighn.local.drupal.uiowa.edu, grad, and admissions and everything works!

**Pull request merged**